examples/iife/index.html

```diff
@@ -73,37 +81,17 @@
 </head>
 <body>
   <main role="main">
-    <h1>MARS Ads Preview - MAJC Edition</h1>
+    <h1>Moz Ads Examples Page, powered by MAJC</h1>
 
     <h2>Tiles</h2>
 
-    <!-- This is the templated version of the preview page (from Shepherd Preview)
-      This shows how the DOM should end up for tiles after showAd is done
-        // <a href="{{ tile.url }}" target="_blank" rel="noopener noreferrer">
-        <div class="tile">
-          <div class="tile-outer">
-            <img
-              class="tile-image"
-              src="{{ tile.image_url }}"
-              alt="{{ tile.name }}"
-            />
-          </div>
-          <div class="tile-title">
-            <span>{{ tile.name }}</span>
-            <span class="sponsored-label">{{ tile.sponsored }}</span>
-          </div>
-        </div>
-      </a> -->
-
     <div class="tiles">
       <div id="placement-1"></div>
     </div>
 
-    <h2>SPOCs</h2>
-    <div class="spocs"></div>
-
     <h2>Rectangles</h2>
     <div class="rectangles">
+      <div id="placement-4"></div>
```

Comment:
nit: should these be in order?

Reply:
Yes, great catch!

And even better, it's a clue about whatever is going wrong with the click nagivation for Mobile Chrome with the Skyscraper. Skyscraper was the last `div` before, and now that I've moved the Medium Rectangle to be the last `div`, Playwright is having trouble clicking on the rectangle as well.  

When I look at what Playwright is doing in the headed view, it seems like Playwright is trying to scroll down but can't get the last element in that div to come into view enough to be clickable... going to try giving the IIFE page a nicer, responsive layout and see if that resolves the issue ...

Reply:
Even with this clue, haven't made progress on this.

Looks like [someone might have had a related issue](https://github.com/microsoft/playwright/issues/33626) but there is no resolution, tomorrow I'll dig into their source code and see if I can notice anything in common with ours related to flex settings ... 

Lmk if you have any ideas, otherwise I'll disable the tests again to get this merge-able and I'll try again in the follow up PR